packages/orbit-components/.storybook/main.ts

```diff
@@ -1,28 +1,34 @@
 /* eslint-disable no-param-reassign */
-import path from "path";
+import path, { dirname, join } from "path";
 import type { StorybookConfig } from "@storybook/react-webpack5";
 
+function getAbsolutePath(value: string): any {
+  return dirname(require.resolve(join(value, "package.json")));
+}
+
 const config: StorybookConfig = {
   staticDirs: [path.resolve(__dirname, "../static")],
   stories: ["../src/**/*.stories.*"],
-  framework: "@storybook/react-webpack5",
+  framework: getAbsolutePath("@storybook/react-webpack5"),
+
   addons: [
-    "@storybook/addon-knobs",
-    "@storybook/addon-a11y",
-    "storybook-addon-pseudo-states",
+    getAbsolutePath("@storybook/addon-knobs"),
+    getAbsolutePath("@storybook/addon-a11y"),
+    getAbsolutePath("storybook-addon-pseudo-states"),
     {
-      name: "@storybook/addon-essentials",
+      name: getAbsolutePath("@storybook/addon-essentials"),
       options: {
         controls: false,
       },
     },
     {
-      name: "@storybook/addon-styling",
+      name: getAbsolutePath("@storybook/addon-styling"),
```

Comment:
I've run into another issue - `getAbsolutePath` was added by upgrade guide (but only to the kbos, a11y and pseudo states). I added it to both addon-styling and addon-essentials because ESLint was complaining ("The @storybook/addon-styling is not installed in orbit/. Did you forget to install it or is your package.json in a different location?")

But if it's kept like this, I cannot run storybook on localhost 🤔 But there seems to be no issue in staging.